package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/apps-cli",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
@harshithad0703 , minor version bump as it's enhancement

Reply:
version bump to 1.3.0